news/2017-12-11-cast-cisl.html

```diff
@@ -0,0 +1,134 @@
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <header class="floe-header">
+                <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <div class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>FLOE Work to Be a Part of the Center on Inclusive Software for Learning</h3>
+                <p class="floe-date">December 11, 2017</p>
+                <p>
+                    We're pleased to announce that tools and knowledge from FLOE work on inclusive open education will be a part of to the newly-announced <a href="http://www.cast.org/whats-new/news/2017/center-for-inclusive-software-for-learning.html#.Wi7t9xNSwUE">Center on Inclusive Software for Learning (CISL)</a> as part of the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>'s partnership on the project.
+                </p>
+                <p>
+                    Led by <a href="http://www.cast.org/">CAST</a> and funded by the the U.S. Department of Education's <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html">Office of Special Education Programs</a>, CISL will support the development of an open-source software suite, industry guidelines and suporting research to ensure K-12 students get engaging personalized learning through Open Education Resources.
+                </p>
+                <p>
+                    We look forward to working with CAST and project partner <a href="https://www.sri.com/">SRI International</a> on advancing the state of the art in inclusive open education with FLOE work!
```

Comment:
Would it still make sense if we removed the "work" from the end of the sentence. I'm thinking it's enough to just say FLOE. What do you think?